dev/Repeater/InspectingDataSource.h

```diff
@@ -43,7 +43,9 @@ class InspectingDataSource :
         const winrt::Collections::IObservableVector<winrt::IInspectable>& sender,
         const winrt::Collections::IVectorChangedEventArgs& e);
 
+    bool m_vectorViewInsteadOfVector{ false };
```

Comment:
could we use m_vectorView != nullptr instead of this bool?

Reply:
I've updated the code to get rid of the flag now.